plugins/scaffolder/src/components/fields/OwnedEntityPicker/useOwnedEntities.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  catalogApiRef,
+  loadCatalogOwnerRefs,
+  loadIdentityOwnerRefs,
+} from '@backstage/plugin-catalog-react';
+import { identityApiRef, useApi } from '@backstage/core-plugin-api';
+import { Entity, RELATION_OWNED_BY } from '@backstage/catalog-model';
+import { CatalogListResponse } from '@backstage/catalog-client';
+import { useAsync } from 'react-use';
+import { useMemo } from 'react';
+
```

Comment:
I think this hook should be in `catalog-react` package instead.

Reply:
Did the change. It's now in `catalog-react`